c_fast.c

```diff
@@ -171,9 +170,8 @@ int main() {
     result = get_max_cost_small(costs, 0, 0);
   }
   gettimeofday(&end, NULL);
-  
+
   timersub(&end, &start, &duration);
   ms = duration.tv_sec*1000 + duration.tv_usec/1000;
-  printf("%d LANGUAGE C-fast %llu\n", result, ms);
+  printf("%d LANGUAGE C-fast %lu\n", result, ms);
```

Comment:
That's incorrect on 32-bit architectures (including Aarch32 and X86), on which a long unsigned int is a uint32_t.